### ISSUE TITLE
LUIS 2.0.2 regression - cannot import LUIS JSON

### ISSUE THREAD

**Issue by srozga** [Microsoft/botbuilder-tools#307]
**Describe the bug**
I had a script workflow that takes a ludown file, uses `ludown `to create a new `luis.json` out of it and then imports the new version into an existing LUIS app. Used to work just fine. When I run `luis import version` now, I get the error: `Missing phraselist features field.`

**To Reproduce**
Assume this is my lu file:
```
> ! Automatically generated by [LUDown CLI](https://github.com/Microsoft/botbuilder-tools/tree/master/Ludown), Tue Jul 17 2018 11:57:24 GMT-0400 (Eastern Daylight Time)

> ! Source LUIS JSON file: bad.json

> ! Source QnA TSV file: Not Specified


> # Intent definitions

## IntentOne
- this is one {a=one} {b=cat}


> # Entity definitions

$a:simple

$b:simple


> # PREBUILT Entity definitions


> # Phrase list definitions


> # List entities


```
run `ludown parse ToLuis --in test.lu` to create test.json

The test.json is:
```
{
  "intents": [
    {
      "name": "IntentOne"
    }
  ],
  "entities": [
    {
      "name": "a",
      "roles": []
    },
    {
      "name": "b",
      "roles": []
    }
  ],
  "composites": [],
  "closedLists": [],
  "regex_entities": [],
  "model_features": [],
  "regex_features": [],
  "utterances": [
    {
      "text": "this is one one cat",
      "intent": "IntentOne",
      "entities": [
        {
          "entity": "a",
          "startPos": 12,
          "endPos": 14
        },
        {
          "entity": "b",
          "startPos": 16,
          "endPos": 18
        }
      ]
    }
  ],
  "patterns": [],
  "patternAnyEntities": [],
  "prebuiltEntities": [],
  "luis_schema_version": "3.0.0",
  "versionId": "0.1",
  "name": "test",
  "desc": "",
  "culture": "en-us"
}
```

Now have an app handy and run import on it. In my case:
`luis import version --in test.json`

My result is:
![image](https://user-images.githubusercontent.com/1940385/43807876-56ecc5b8-9a78-11e8-8bbe-0ef32482cf91.png)

**Comment by vishwacsena** [comment]
@srozga - can you confirm your luis cli version? I presume you are on 2.0.2? I believe this is a regression introduced in LUIS cli 2.0.2. As a workaround, can you downgrade the LUIS CLI to 1.0.35 via 

```bash
> npm i -g luis-apis@1.0.35
```

I verified that luis import works fine on that version. We will continue to use this bug to track a fix for the LUIS CLI.

@tomlm - FYI

**Comment by vishwacsena** [comment]
@tomlm - luis import application as well as luis import version are broken in 2.0.2. I verified that the JSON in the example above works fine with 1.0.35 and also imports successfully in the portal.

**Comment by tomlm** [comment]
There is a regression in the API.  if you change the regex_features and regex_entities to be regexFeatures and regexEntities it works. I am working with LUIS team to resolve.

**Comment by tomlm** [comment]
fixed in version 2.0.3
